buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -26,13 +26,22 @@ object Libraries {
         const val constraintLayout = "1.1.3"
         const val ktx = "1.3.0"
         const val material = "1.1.0"
+        const val koin = "2.1.6"
+        const val lifecycleKtx = "2.2.0"
     }
 
     const val kotlinStdLib     = "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlinVersion"
     const val appCompat        = "androidx.appcompat:appcompat:${Versions.jetpack}"
     const val constraintLayout = "androidx.constraintlayout:constraintlayout:${Versions.constraintLayout}"
     const val ktxCore          = "androidx.core:core-ktx:${Versions.ktx}"
     const val material         = "com.google.android.material:material:${Versions.material}"
+    const val livedataKtx     = "androidx.lifecycle:lifecycle-livedata-ktx:${Versions.lifecycleKtx}"
+
+    object Koin {
+        const val androidCore  = "org.koin:koin-android:${Versions.koin}"
+        const val androidScope = "org.koin:koin-android-scope:${Versions.koin}"
```

Comment:
Do we wanna do Koin scoping in this MVP? if not, this line is not needed.

Reply:
hmm, yes. I am trying to avoid it until I really need it. I deleted this one. good catch 👍